torch/ao/quantization/fx/_lower_to_native_backend.py

```diff
@@ -72,8 +72,11 @@ def _lower_to_native_backend(model: QuantizedGraphModule) -> QuantizedGraphModul
     operator signature so they can be lowered with the same function
     """
     model = _lower_ref_linear_module(model)
-    model.recompile()
     for pattern, replacement in get_fbgemm_patterns_and_replacements():
-        subgraph_rewriter.replace_pattern(model, pattern, replacement)
+        while True:
```

Comment:
should this be an iteration guard, to prevent an infinite loop if a bug sneaks in to `subgraph_rewriter.replace_pattern`?  Something like

```
# eliminate the possibility of an infinite loop
MAX_ITER = 1000
cur_iter = 0
while cur_iter < MAX_ITER:
  ...
  cur_iter += 1
```